### PR TITLE
nlp_resume test config and samples

### PR DESCRIPTION
**Pull request by aNorthSense**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,9 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 
 - [CMS 1500 (08/05)](./health_insurance_claims/cms_1500/cms_1500_08_05.json)
 
+### NLP
```
Comment (ethulia) [inline README.md:215]: Let's call this "HR" instead, we should categorize these by use case rather than the methods that we used to set it up.

**Review by ethulia — CHANGES_REQUESTED** [review]

**Review by ethulia — CHANGES_REQUESTED** [review]
One last change, can we just call it resume rather than nlp_resume?

File: README.md [hunk 2]
```diff
@@ -212,6 +212,9 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 
 - [CMS 1500 (08/05)](./health_insurance_claims/cms_1500/cms_1500_08_05.json)
 
+### HR
+
+- [Resume Example](./resume/resume.json)
```
Comment (ethulia) [inline README.md:217]: Can we please just call this `Resume`?

**Review by ethulia — CHANGES_REQUESTED** [review]

File: HR/config.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "category": "HR",
+    "description": "HR documents, resumes, etc",
+    "display_name": "HR and people documents, such as resume",
+    "featured": {
+      "primary": "nlp_resume",
```
Comment (ethulia) [inline HR/config.json:6]: this needs to get changed to `resume` since we changed the file names.

**Review by ethulia — CHANGES_REQUESTED** [review]

File: HR/config.json (hunk 3 above)
Comment (aNorthSense) [reply HR/config.json:6]: Oh, good catch, doing that now

**Review by ethulia — APPROVED** [review]

**Pull request merged**